### PR TITLE
[docs] add Vale as a documentation linter

### PR DESCRIPTION
**Pull request by odow**
Experimenting again with https://vale.sh.

It found a few typos.

### CODE REVIEW

**Comment by mlubin** [comment]
That's a fair number of typos.

**Comment by odow** [comment]
Maintaining the accept list seems like it could be a pain. But the real issue is that GitHub doesn't have any linting in the web editor, so it's hard to catch these in review.

Anyway. More tooling seems like positive progress if it leads to a higher quality product.

**Review by blegat — APPROVED** [review]
This is nice, I didn't know this existed

**Comment by odow** [comment]
This needs a new tag of MOI so we don't fail on the errors in MOI's docs.

File: docs/styles/Vocab/JuMP-Vocab/accept.txt [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+MOI
+Mosek
+MOSEK
+Pajarito
+PATHSolver
+Penbmi
+puzzlor
+RAPOSa
+Xpress
+
```
Comment (joaquimg) [inline docs/styles/Vocab/JuMP-Vocab/accept.txt:105]: empty?

**Pull request merged**